src/recipe-browser-backend/WeatherForecast.cs

```diff
@@ -0,0 +1,13 @@
+namespace recipe_browser_backend
+{
+    public class WeatherForecast
+    {
+        public DateOnly Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
+
+        public string? Summary { get; set; }
+    }
+}
```

Comment:
Ten kod to jest jakiś wygenerowany?

Reply:
Aaaa tak zapomnialem usunac, to jest autogenerowane wraz z projektem